src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -569,161 +559,133 @@
 
-        "properties": 
-        {
-          "description": "Key/value pairs that provide additional details about the run.",
+        "properties": {
+          "description": "Key/value pairs that provide additional information about the run.",
           "type": "object",
-          "default": {}
+          "default": { }
         },
 
-        "tags": 
-        {
-          "description": "A unique set of strings that provide additional information for the run.",
+        "tags": {
+          "description": "A set of distinct strings that provide additional information about the run.",
           "type": "array",
-          "items":
-          {
+          "items": {
             "type": "string",
-            "default": []
-          }
-        }
-      }
-    },
-    "runLog": 
-    {
-      "additionalProperties": false,
-      "type": "object",
-      "properties": {
-        "tool":
-        {
-          "description": "Information about the tool or tool pipeline that generated the results in this log. A results log can only contain results produced by a single tool or tool pipeline. A results log can aggregate results from multiple tool log files, as long as context around the tool run (tool command-line arguments and the like) is identical for all aggregated files.",
-          "$ref": "#/definitions/tool"
-        },
-
-        "run": 
-        {
-          "description": "A run object describes the invocation of the static analysis tool that produced the results specified in the containing runLog object.",
-          "$ref": "#/definitions/run"
-        },
-
-        "rules": 
-        {
-          "description": "An array of rule descriptor objects that describe all rules associated with an analysis tool or a specific run of an analysis tool.",
-          "type": "array",
-          "minItems": 1,
-          "items":
-          {
-            "$ref": "#/definitions/ruleDescriptor"
-          }
-        },
-
-        "results": 
-        {
-          "description": "The set of results contained in an SARIF log. The results array can be omitted when a runLog is merely exporting rules metadata. It must be present (but may be empty) in the event that a log file represents an actual scan.",
-          "type": "array",
-          "minItems": 0,
-          "items":
-          {
-            "$ref": "#/definitions/result"
+            "default": [ ]
           }
         }
       },
-      "required": ["tool"]
+      "required": [ "tool" ]
     },
     "tool": 
     {
-      "description": "Information about the tool or tool pipeline that generated the results in this log.",
+      "description": "Describes that static analysis tool that was run.",
       "additionalProperties": false,
       "type": "object",
       "properties": {
         "name": 
         {
-          "description": "The name of the tool or tool pipeline that generated the results in this log, e.g., FxCop.",
+          "description": "The name of the tool.",
```

Comment:
Nah.

---
In reply to: [59471244](https://github.com/Microsoft/sarif-sdk/pull/178#discussion_r59471244) [](ancestors = 59471244)